ui/src/router/components/form/validation/schema.js

```diff
@@ -72,7 +74,8 @@ const validateDanglingRoutes = function (items) {
       errors.push(
         this.createError({
           path: `${this.path}[${idx}].id`,
-          message: "This route should be removed since they have no Traffic Rule(s) associated and will never be called.",
+          message:
+            "This route should be removed since they have no Traffic Rule(s) associated and will never be called.",
```

Comment:
Not a change introduced in this PR but I think it should be `since it has no` instead of `since they have no`.